evm_loader/lib/src/rpc/db_call_client.rs

```diff
@@ -94,4 +94,8 @@ impl Rpc for CallDbClient {
     async fn get_slot(&self) -> ClientResult<Slot> {
         Ok(self.slot)
     }
+
+    fn supports_features(&self) -> bool {
+        false
```

Comment:
Features should be supported by the ClickHouse DB in the future.
I don't want to remember to change the neon-api code when this happens.